code/game/gamemodes/events/biomass.dm

```diff
@@ -27,13 +27,11 @@
 	if (!W || !user || !W.type)
 		return
 
-	switch(W.type)
+	switch(W.type) //This is absolutely terrible

```

Comment:
Change this to

    if(prob(W.is_sharp() * 100) || W.is_hot())
        qdel(src)